.github/workflows/ci.yml

```diff
@@ -111,18 +111,44 @@ jobs:
           python -m pip install -r requirements-ci.txt
       - name: analyze example session
         timeout-minutes: 5
+        if: always()
```

Comment:
this is not needed here (we set it for consequent steps)

Reply:
I added it because we had a failure (no stats file) but the session was already created. The idea is that this step can fail but we can still upload a possibly insightful session in the later steps.